### PR TITLE
Update fail2ban.markdown

### PR DESCRIPTION
**Pull request by mplinuxgeek**
Updated using more consistent paths for the filter and jail configuration.
Added additional checks to help confirm everything working as expected.
Added steps to unblock an IP address.
Updated jail conf file includes email alerts.
Removed the syslog section, this step is not required for fail2ban thus optional and only required if you want HA logs to be sent to syslog, should be covered by another page.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_cookbook/fail2ban.markdown [hunk 1]
```diff
@@ -2,103 +2,121 @@
...
-May 24 20:58:02 homeauto fail2ban.jail[15217]: INFO Jail 'sshd' started
-May 24 20:58:02 homeauto fail2ban.jail[15217]: INFO Jail 'hass-iptables' started
+Restart fail2ban:
+`sudo systemctl restart fail2ban`
+
+Confirm fail2ban is running:
+`sudo systemctl status fail2ban`
+
+Check that the ha jail is active:
+```sudo fail2ban-client status
```
Comment (fabaff) [inline source/_cookbook/fail2ban.markdown]: Dito

File: source/_cookbook/fail2ban.markdown [hunk 2]
```diff
@@ -2,103 +2,121 @@
...
+```sudo yum install epel-release
+sudo yum install -y fail2ban```
 
-Then make sure logging is enabled in your `configuration.yaml` file for your Home Assistant instance:
+For other package managers use the appropriate commands.
 
-```yaml
-logger:
+First, enable http.ban logging in `configuration.yaml` file for your Home Assistant instance:
+```logger:
```
Comment (fabaff) [inline source/_cookbook/fail2ban.markdown]: This needs be reverted. `yaml` is for the syntax highlighting.

File: source/_cookbook/fail2ban.markdown [hunk 3]
```diff
@@ -2,103 +2,121 @@
...
 
-This is a quick guide on how to setup fail2ban for Home Assistant. This was originally in the [forum](https://community.home-assistant.io/t/is-there-a-log-file-for-invalid-logins-blocking-hackers/2892) but I created this here for people.
+This is a quick guide on how to setup fail2ban for Home Assistant. Contains extracts from [forum](https://community.home-assistant.io/t/is-there-a-log-file-for-invalid-logins-blocking-hackers/2892).
 
-First install `fail2ban`. On Debian/Ubuntu this would be `apt-get install fail2ban`. On other distros you can google it.
+Install `fail2ban`. 
+Debian/Ubuntu:
+`sudo apt-get install fail2ban`.
+CentOS/RHEL/Fedora (use dnf on Fedora):
+```sudo yum install epel-release
```
Comment (fabaff) [inline source/_cookbook/fail2ban.markdown]: `epel-release` is not needed for Fedora.

**Review by fabaff — CHANGES_REQUESTED** [review]
The file needs to follow the markdown standards.

File: source/_cookbook/fail2ban.markdown (hunk 3 above)
Comment (mplinuxgeek) [reply source/_cookbook/fail2ban.markdown]: Fixed

File: source/_cookbook/fail2ban.markdown (hunk 2 above)
Comment (mplinuxgeek) [reply source/_cookbook/fail2ban.markdown]: Fixed

File: source/_cookbook/fail2ban.markdown (hunk 1 above)
Comment (mplinuxgeek) [reply source/_cookbook/fail2ban.markdown]: Fixed

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**